src/Microsoft.Health.Dicom.SqlServer/Registration/DicomServerBuilderSqlServerRegistrationExtensions.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.Health.Dicom.Core.Features.Store;
 using Microsoft.Health.Dicom.Core.Registration;
 using Microsoft.Health.Dicom.SqlServer.Features.ChangeFeed;
+using Microsoft.Health.Dicom.SqlServer.Features.CustomTag;
```

Comment:
Yes, since the SqlCustomTagStore is under this namespace!

---
In reply to: [577198848](https://github.com/microsoft/dicom-server/pull/542#discussion_r577198848) [](ancestors = 577198848)